pkg/api/kubegraph/nodes/nodes.go

```diff
@@ -130,8 +130,10 @@ func EnsurePodTemplateSpecNode(g osgraph.MutableUniqueGraph, ptSpec *kapi.PodTem
 		},
 	).(*PodTemplateSpecNode)
 
-	podSpecNode := EnsurePodSpecNode(g, &ptSpec.Spec, ptSpecName)
-	g.AddEdge(ptSpecNode, podSpecNode, osgraph.ContainsEdgeKind)
+	if ptSpec != nil {
```

Comment:
I still maintain that `ptSpec` should never be `nil` when this method is called.  Can you chase the offender?

You can't reasonable ensure a Node to a `nil` element.